src/sentry/replays/usecases/ingest/__init__.py

```diff
@@ -261,6 +261,23 @@ def recording_post_processor(
             message.replay_id,
             parsed_segment_data,
         )
+
+        # Log # of rrweb events to bigquery.
+        logger.info(
+            # Logging to the sentry.replays.slow_click namespace because
+            # its the only one configured to use BigQuery at the moment.
+            #
+            # NOTE: Needs an ops request if we want to create a new dataset.
```

Comment:
What's the ops request comment mean? @cmanallen

Reply:
You can ignore it.  Its a note about the destination table name: `"sentry.replays.slow_click"`.  It can be removed.